### ISSUE TITLE
USER ISSUE: Exception when filling full or nearly full inventory

### ISSUE THREAD

**Issue by lemarkis** [StrangeLoopGames/EcoIssues#5588]
**Version:** 0.7.2.0 beta staging-f6bf50db

**Steps to Reproduce:**
1. Carry some blocks
2. Right click on a (nearly) full stockpile to put block in

**Expected behavior:**
It should have pop the message "No room in destination inventory" or fill only empty room.

**Actual behavior:**
The following error message popped up:
```
Caught exception:

<size=60,00%>Caught exception invoking RPC PlayerInteract on Player!

La référence d'objet n'est pas définie à une instance d'un objet.


Stack:
   à Eco.Gameplay.Items.Inventory.MoveAsManyItemsAsPossible(Inventory destination, Func`2 sourceStackPredicate, User user)
   à Eco.Gameplay.Components.StorageComponent.OnActLeft(InteractionContext context)
   à Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   à Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   à Eco.Gameplay.Players.Player.PlayerInteract(InteractionInfo info)



Stack:
   à Eco.Shared.Networking.RPCManager.TryInvoke(Object controller, String methodname, BSONObject bsonArgs, Object& result)
   à Eco.Shared.Networking.RPCManager.InvokeOn(BSONObject bson, Object controller, String methodname)
   à Eco.Shared.Networking.RPCManager.HandleReceiveRPC(INetClient client, BSONObject bson)
   à Eco.Shared.Networking.NetPort.ProcessBuffer()
   à Eco.Shared.Networking.NetPort.Update()
   à Eco.Plugins.Networking.Clients.Client.<.ctor>b__56_0()
</size>
```

**Comment by destruktoid** [comment]
Similar error when taking items out of stockpile. Steam version 7.2.0 beta

Steps:
1. Have a less than full stack of items in hand
2. Left click to take items out.
```
Caught exception:

<size=60.00%>Caught exception invoking RPC PlayerInteract on Player!

Object reference not set to an instance of an object.


Stack:
   at Eco.Gameplay.Items.Inventory.MoveAsManyItemsAsPossible(Inventory destination, Func`2 sourceStackPredicate, User user)
   at Eco.Gameplay.Components.StorageComponent.OnActLeft(InteractionContext context)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Players.Player.PlayerInteract(InteractionInfo info)



Stack:
   at Eco.Shared.Networking.RPCManager.TryInvoke(Object controller, String methodname, BSONObject bsonArgs, Object& result)
   at Eco.Shared.Networking.RPCManager.InvokeOn(BSONObject bson, Object controller, String methodname)
   at Eco.Shared.Networking.RPCManager.HandleReceiveRPC(INetClient client, BSONObject bson)
   at Eco.Shared.Networking.NetPort.ProcessBuffer()
   at Eco.Shared.Networking.NetPort.Update()
   at Eco.Plugins.Networking.Clients.Client.<.ctor>b__56_0()
</size>
```

**Comment by SeanAtr0n** [comment]
Fixed in 7.2.1
